themes/jquery/functions.php

```diff
@@ -251,3 +251,36 @@ function jq_image_posted_on() {
 
 	return $classes;
 } );
+
+/**
+ * Content Security Policy
+ */
+function jq_content_security_policy() {
+  $nonce = wp_create_nonce( JQUERY_LIVE_SITE );
+  $policy = array(
+    'default-src' => "'self'",
+    'script-src' => "'self' 'nonce-$nonce' code.jquery.com",
```

Comment:
This nonce can be used by any site in case an inline script or style is needed in the theme. We may not need this, but this is how it could be done.